Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'puma', '~> 3.11'
 gem 'fast_jsonapi'
 # gem 'active_model_serializers'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.5'
+# gem 'jbuilder', '~> 2.5'
```

Comment:
was able to accomplish serializing revenue with a revenue poro so jbuilder now not needed